.env

```diff
@@ -85,3 +85,5 @@ REACT_APP_FEEDBACK_ENABLED_DEV=false
 # Amplitude
 # REACT_APP_AMPLITUDE_KEY=
 
+# Launch darkly
+REACT_APP_LD_CLIENT_KEY=643681f370ea6d135128b2ce
```

Comment:
could we include DARKLY instead of LD in the name so is easier to know what it is about without comments (note in GH secrets we won't see the comment)

`REACT_APP_DARKLY_KEY`?

Reply:
Good point, will use `REACT_APP_LAUNCH_DARKLY_KEY`